.github/labels.yml

```diff
@@ -0,0 +1,129 @@
+- name: "2 - Working"
+  color: "CCCCCC"
+  description: "A user or team member has started working on the issue"
+- name: "3 - Review"
+  color: "CCCCCC"
+  description: "Code has been added, and is available for review as a pull request"
+- name: "4 - Done"
+  color: "CCCCCC"
+  description: "Code has been added to the repository, and has been reviewed by a team member"
+- name: "5 - Released"
+  color: "CCCCCC"
+  description: "The issue has been resolved, and released to the public for consumption"
+- name: "ADD AUTO TESTS"
+  color: "8E44AD"
+  description: "Things that typically go to Test-Kitchen - once completed, IN REGRESSION TEST SUITE label added"
+- name: "ADD MANUAL TESTS"
+  color: "8E44AD"
+  description: "Manual tests has not been added for issue - once completed, IN MANUAL TEST SUITE label added"
+- name: "API"
+  color: "207DE5"
+  description: "Issues that handles API changes, not necessarily visible to end users"
+- name: "Bad Pull Request"
+  color: "EB6420"
+  description: "The pull request opened was not accepted, didn't follow the procedures, or was missing information"
+- name: "Blocked - External"
+  color: "5319E7"
+  description: "The issue or pull request is not fixable without a change in a upstream/external library"
+- name: "BreakingChange"
+  color: "5319E7"
+  description: "The issue will introduce backwards incompatible changes"
+  aliases: ["Breaking Change"]
+- name: "Bug"
+  color: "D73A4A"
+  description: "Issues where something had happened which was not expected or intended"
+  aliases: ["bug"]
+- name: "BuildAutomation"
+  color: "0052CC"
+  description: "Issues for changes to the build system, without functional changes needed to the project"
+  aliases: ["Build Automation", "Build"]
+- name: "CoreTeam"
+  color: "F7C6C7"
+  description: "Issues that will only be accepeted from the core Chocolatey Team"
+- name: "Customer"
+  color: "006B75"
+  description: "Bug reported or feature requested by a licensed customer of Chocolatey"
+- name: "Detail Required"
+  color: "9C772F"
+- name: "Documentation"
+  color: "BFE5BF"
+  description: "Issues for changes that only need to change documentation"
+  aliases: ["documentation"]
+- name: "Duplicate Issue"
+  color: "EB6420"
+  description: "Issues that are duplicates of other issues"
+  aliases: ["duplicate", "duplicates"]
+- name: "Enhancement"
+  color: "0052CC"
+  description: "Issues that enhances existing functionality, or adds new features"
+  aliases: ["enhancement"]
+- name: "Feature"
+  color: "5319E7"
+- name: "Hacktoberfest"
+  color: "FB9904"
+  description: "Issues opened during hacktoberfest"
+- name: "hacktoberfest-accepted"
+  color: "1D76DB"
+  description: "Pull Requests opened during hacktoberfest, and has been accepted by a team member"
+- name: "HasDependency"
+  color: "8E44AD"
+  description: "This issue has a related issue that would need to be addressed before this issue can be closed."

```

Comment:
Some descriptions have a `.` at the end, but others don't.  Should we make them consistent?